src/app/InteractionModelEngine.h

```diff
@@ -343,5 +343,8 @@ CHIP_ERROR ReadSingleClusterData(const Access::SubjectDescriptor & aSubjectDescr
  */
 CHIP_ERROR WriteSingleClusterData(const Access::SubjectDescriptor & aSubjectDescriptor, ClusterInfo & aClusterInfo,
                                   TLV::TLVReader & aReader, WriteHandler * apWriteHandler);
+
+bool IsClusterDataVersionAllowed(const EndpointId & aEndpointId, const ClusterId & aClusterId, const DataVersion & aDataVersion);
```

Comment:
Allowed in what sense?  What is this predicate really answering?  Needs documentation.

And please don't pass small (i.e. fits in a register) integers by const reference?  There's no reason not to pass them by value.

Reply:
Updated documentation, thanks